listenbrainz/webserver/static/js/src/LastFMImporter.tsx

```diff
@@ -151,7 +150,7 @@ export default class LastFmImporter extends React.Component<
     }
   }
 
-  async getPage(page: number) {
+  async getPage(page: number, retries: number) {
     /*
      * Fetch page from Last.fm
```

Comment:
arg documentation, what is page and retries.

Reply:
Done.